tools/perf/perf_run.py

```diff
@@ -187,15 +188,27 @@ def run_ts_trt(model, input_tensors, params, precision, batch_size):
 
     iters = params.get("iterations", 20)
 
-    record_perf(
-        model,
-        "Torchscript",
-        input_tensors,
-        precision,
-        iters,
-        batch_size,
-        compile_time_s,
-    )
+    if params.get("enable_cuda_graph", False):
+        with torchtrt.runtime.enable_cudagraphs(model) as cudagraphs_module:
+            record_perf(
+                cudagraphs_module,
+                "Torchscript",
+                input_tensors,
+                precision,
+                iters,
+                batch_size,
+                compile_time_s,
+            )
+    else:
+        record_perf(
+            model,
+            "Torchscript",
+            input_tensors,
+            precision,
+            iters,
+            batch_size,
+            compile_time_s,
+        )
```

Comment:
This option doesn't apply to torchscript, it is only for Torch-TRT (dynamo/torch.compile backend)